content/blog/botan-first-milestone.md

```diff
@@ -0,0 +1,248 @@
+
+Now we are hard at work on getting the first release version of `botan` ready too, and could use your feedback!
+
+## What is Botan?
+
+Botan is an comprehensive, open-source, BSD-licenced, C++ cryptography library with a stable C API. It offers a broad variety of functionality and algorithms, including **post-quantum cryptography**, is developed and maintained by an active community, and has been [audited][botan audit] in the past.
+
+By binding to Botan, we have solved a significant problem of providing much of the necessary 'cryptographic kitchen sink' via a suitably performant, suitably licensed, open-source library. Furthermore, we do this without imposing a large maintenance burden on the Haskell community, as we are not required to maintain the Botan cryptography library itself, only the bindings to it.
+
+## A call for users
+
+Do you use one of the following libraries?
+
+- `crypton` / `cryptonite`
+- `libsodium`
+- `saltine`
+
+That's where you come in! We'd like you to consider giving [botan][botan github] a try!
+
+`botan-low` is surprisingly viable out-of-box, and `botan` will be having it's own `0.0.1` release soon enough. We are interested in seeing how they perform in the real world, and we can only do that with users - we can't listen to your feedback if there isn't any!
+
+Or perhaps you may be a user of a library with one of these buried deep in the dependencies?
+
+- `amazonka-core`
+- `hpack`
+- `scotty`
+- `servant`
+- `stack`
+- `tls`
+- `x509`
+- `warp`
+- `websockets`
+
+There's some pretty popular libraries on that list there, and their list of transitive dependents is quite large. These libraries sit at the root of a lot of production haskell code, and anything involving networking, APIs, and the internet is likely to depend on at least one of them. 
+
+Would it surprise you that they are all directly or indirectly dependent on `crypton`, which contains unaudited C implementations that must be maintained by the Haskell community as a fork after `cryptonite` was abandoned by its original author.
+
+This is a problem, and we're trying to change that with bindings to `botan`.
+
+Having a solid, reliable, and well-maintained cryptography libraries is a huge benefit to the Haskell ecosystem in general, and is essential for most commercial and industrial use cases.
+
+What are you using cryptography for? Let us know in the comments or with upvotes.
+
+## Who are we?
+
+I am Leo Dillinger, a member of [Haskell Cryptography Group][haskell cryptography group], and I am working with the [Haskell Foundation][haskell foundation] to develop free and open-source software for you and the Haskell ecosystem.
+
+## What are our goals?
+
+We seek to provide trusted, open-source cryptography solutions to you. Much of the existing Haskell cryptography ecosystem is aging, unmaintained and unaudited, or very limited in scope, we are seeking to improve that.
+
+Today, we're here to tell you about what is coming next.
```

Comment:
It's a bit redundant, the cryptography group already introduced itself in a previous blog post. 

You should however introduce yourself much earlier in the blog post, possibly in the first sentence.